kratos/processes/from_json_check_result_process.h

```diff
@@ -0,0 +1,921 @@
+//                         auto& p_table = r_table_vector_vector_vector[k][j][i];
+//                         KRATOS_ERROR_IF(p_table == nullptr) << "Table not created" << std::endl;
+//                     }
+//                 }
+//             }
+//         }
+//     #endif
+    }
+
+    /**
+     * @brief This method retrieves the variable database
+     * @param rVariable The variable to be retrieved
+     * @tparam TVariableType The variable type considered
+     */
+    template<class TVariableType>
+    VariableDatabase& GetVariableData(const TVariableType& rVariable)
+    {
+        const auto it = find(rVariable.Key());
+        if (it != end()) {
+            return it->second;
+        } else {
+            KRATOS_ERROR << "Not allocated Variable: " << rVariable.Name() << std::endl;
+        }
+    }
+
+    /**
+     * @brief This method retrieves the variable database
+     * @param rVariable The variable to be retrieved
+     * @tparam TVariableType The variable type considered
+     */
+    template<class TVariableType>
+    const VariableDatabase& GetVariableData(const TVariableType& rVariable) const
+    {
+        const auto it = find(rVariable.Key());
+        if (it != end()) {
+            return it->second;
+        } else {
+            KRATOS_ERROR << "Not allocated Variable: " << rVariable.Name() << std::endl;
+        }
+    }
+
+//     /**
+//      * @brief This method retrieves the interpolated value from the database
+//      * @param rVariable The variable to be retrieved
+//      * @param EntityIndex The index of the entity (not the entity Id, the index in the database)
+//      * @param Time The time value to be retrieved
+//      * @param ComponentIndex The component index of the vector/array
+//      * @param GPIndex The Gauss point index
+//      * @tparam TVariableType The variable type considered
+//      */
+//     template<class TVariableType>
+//     const double GetValue(
+//         const TVariableType& rVariable,
+//         const IndexType EntityIndex,
+//         const double Time,
+//         const SizeType ComponentIndex = 0,
+//         const SizeType GPIndex = 0
+//         ) const
+//     {
+//         return GetVariableData(rVariable).GetValue(EntityIndex, Time, ComponentIndex, GPIndex);
+//     }
+
+    /**
+     * @brief This method set the values into the tables
+     * @param rValuesX The values of the X axis
+     * @param rValuesY The values of the Y axis
+     * @param rVariable The variable to be retrieved
+     * @param EntityIndex The index of the entity (not the entity Id, the index in the database)
+     * @param ComponentIndex The component index of the vector/array
+     * @param GPIndex The GP index
+     * @tparam TVariableType The variable type considered
+     */
+    template<class TVariableType>
+    void SetValues(
+        const Vector& rValuesX,
+        const Vector& rValuesY,
```

Comment:
It is most probably a stupid question, but shouldn't it be necessary to define a similar one also with `rValuesZ` for 3D cases?

Reply:
These are the values of a table, x is the first column, y the second

Reply:
Correct. It was a stupid question.